docs/release_notes/release_notes-0.15.0.rst

```diff
@@ -13,12 +13,18 @@ Key Features and Updates
   * FIX-#4385: Get rid of `use-deprecated` option in `pip` (#4386)
   * FIX-#3527: Fix parquet partitioning issue causing negative row length partitions (#4368)
   * FIX-#4330: Override the memory limit to start ray 1.11.0 on Macs (#4335)
+  * FIX-#4407: Align `insert` function with pandas in case of numpy array with several columns (#4408)
```

Comment:
@amyskov, the diff now includes the changes that have been merged earlier into master. Please correct the changes that should be in this branch.